src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Serialization/ObjectWriter.cs

```diff
@@ -129,6 +129,19 @@ public void Dispose()
     public void WriteUInt16(ushort value) => _writer.Write(value);
     public void WriteString(string? value) => WriteStringValue(value);
 
+    public Stream BaseStream => _writer.BaseStream;
+
+    public void Reset(bool includeValidationBytes = true)
```

Comment:
hrmm... why is this optional.  that seems... onfusing.

Reply:
no one seems to ever call this with a value.  so just remove and always include the bytes?

Reply:
Yeah, probably shouldn't have worried about supporting that case